core/src/test/resources/sql/planner.iq

```diff
@@ -59,9 +59,9 @@ EnumerableNestedLoopJoin(condition=[IS NOT DISTINCT FROM($0, $1)], joinType=[sem
     EnumerableValues(tuples=[[{ 0 }, { 1 }]])
 !plan
 
-!set reset-rules ""
+!set planner-rules original
 
-# Test reset-rules can reset init rules
+# Test that rules set by "!set planner-rules original" are sticky
```

Comment:
"sticky" means that they can be reused across multiple tests. 
this is not what you are testing here.
you are really testing that this brings back the original rules (to some degree)

Reply:
Sorry, I fixed this. When I was developing, I found that multiple SQL statements would be affected by the changed rules if I didn't reset them. So I specifically verified this problem when debugging.